firestore/integration_test_internal/src/validation_test.cc

```diff
@@ -350,8 +515,12 @@ TEST_F(ValidationTest, WritesMayContainIndirectlyNestedLists) {
 #endif  // defined(FIREBASE_USE_STD_FUNCTION)
 }
 
-// TODO(zxu): There is no way to create Firestore with different project id yet.
-TEST_F(ValidationTest, WritesMustNotContainReferencesToADifferentDatabase) {}
+// TODO(b/136012313): batches and transactions don't currently throw.
+// TEST_F(ValidationTest, WritesMustNotContainReferencesToADifferentDatabase) {
```

Comment:
Consider uncommenting this test, but skipping it via the `GTEST_SKIP()` macro. This will at least highlight in the test result count that a test has been skipped. Leaving it commented here makes it less visible.

There is precedence for this too, like all of the macros in https://github.com/firebase/firebase-cpp-sdk/blob/8ad99ec49ff5d01e909d5cedc5366cf5325460ec/testing/test_framework/src/firebase_test_framework.h.

Reply:
Thanks, great suggestion.